docs/version0.46.md

```diff
@@ -72,6 +73,10 @@ The extended HCR feature is deprecated in this release and will be removed in a
 
 When the `jcmd` tool sends a command to a running VM, the command might throw the `Socket Exception` error on Windows platform. Instead of failing the attaching request, you can specify the number of times the tool retries attaching to the target VM with the new system property, [`-Dcom.ibm.tools.attach.retry`](dcomibmtoolsattachretry.md).
 
+### `-Xtgc:allocation` report includes core allocation cache stats per thread
+
+The [`-Xtgc:allocation`](xtgc.md#allocation) option prints thread-specific allocation cache (TLH) stats in addition to the cumulative allocation stats.
```

Comment:
Wouldn't it be better not to abbreviate "statistics"?

Reply:
Done. Thanks!

Reply:
Any other changes? Can this be merged?